CoEpi/ui/alerts/AlertsViewModel.swift

```diff
@@ -13,9 +13,8 @@ class AlertsViewModel {
     init(alertRepo: AlertRepo) {
         self.alertRepo = alertRepo
 
-        title = alertRepo.alerts
-            .map { AlertsViewModel.formatTitleLabel(count: $0.count) }
-            .startWith(AlertsViewModel.formatTitleLabel(count: 0))
+        title = alertRepo.alerts.map { _ in AlertsViewModel.formatTitleLabel() }
```

Comment:
I'd remove this and set the translation directly in the view controller. The title doesn't depend on the alerts anymore.